app/controllers/sites_controller.rb

```diff
@@ -25,12 +25,12 @@ def robots
   def set_places_to_get_computer_access
     tag = Tag.find_by(slug: 'computers')
 
-    @places_to_get_computer_access = Partner.for_site_with_tag(current_site, tag)
+    @places_to_get_computer_access = Partner.for_site(current_site).with_tags(tag)
```

Comment:
Do we need to add the visible flag back in to this query?

Reply:
Good thinking - looks like the `for_site` method does check visibility too tho. Which does raise the question if it _should_ - but poss a tech debt question for antoher day?